sdks/python/apache_beam/io/external/xlang_kafkaio_it_test.py

```diff
@@ -138,6 +143,44 @@ def test_kafkaio_null_key(self):
       self.run_kafka_write(pipeline_creator)
       self.run_kafka_read(pipeline_creator, None)
 
+  @pytest.mark.uses_gcp_java_expansion_service
```

Comment:
This in combine with the kafka dependency in gcp expansion service work in a way that inject kafka dependency and tests into existing gcp_io_xlang test suite. I would suggest think about a more clean way to do this.

e.g. consider another mark for kafka test. It does not use PTransforms in :sdks:java:io:google-cloud-platform project